### ISSUE TITLE
YouTube Lectures

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci040#2]
The CS40 playlist on youtube is at: https://www.youtube.com/playlist?list=PLSNWQVdrBwoZugWrBEGivQAcUVSP-pRRk

In general, I'll upload the videos to this playlist within a few hours of class.

**Comment by mikeizbicki** [comment]
Today's lectures have finished uploading.  In the future, I won't post an announcement when the upload finishes, the videos will just be waiting in the playlist for you.

**Comment by mikeizbicki** [comment]
It looks like the lecture video for Monday 31 Aug was corrupted somehow, and so youtube is not showing the video correctly.

This video covered lessons 1-5 of our HTML/CSS textbook: https://learn.shayhowe.com/html-css/  For those of you who were not able to attend the lecture live, I recommend reading those chapters of the textbook.  I'm also happy to schedule extra office hours (at a time convenient for your timezone) to answer any questions you have about the material personally.

Due to the inconvenience this has caused, I'm extending the deadline for the first homework assignment by 2 days until Tuesday 8 September.

If you feel like you need additional accommodations due to this technical problem, feel free to email me and we'll work something out.  (Email, instead of a github issue, is acceptable for private communications about accommodations issues.)

**Comment by mikeizbicki** [comment]
Well... crap.  The lecture video for Monday 6 Sept was also corrupted, and so it's not available on youtube.  In both cases, my zoom crashed during the middle of the session, and that's what I believe caused the video to be corrupted.  I realize this is incredibly frustrating for you, and doubly so for those of you in different timezones who were not able to attend the live lecture :(

For those of you who missed the lecture: we covered the basic python concepts that are also covered in the prelecture videos, and we also covered something called "doctests".  We will review the doctest content again in today's lecture, which will hopefully be successfully recorded.

Going forward, I will continue recording the lectures and uploading them when possible.  I don't think there's anything I can do though if zoom continues to corrupt the video files :(

**Comment by adien43** [comment]
Mike could we get access to Wednesday's class recording? Thank you!

**Comment by mikeizbicki** [comment]
I've uploaded the last 15 minutes or so of the lecture to youtube, but zoom crashed and so I'm unable to load the entire lecture again.

In the first part of the lecture, we covered the instructions for the [hw_01](https://github.com/mikeizbicki/cmc-csci040/tree/2020fall/hw_01) assignment.  We then continued to work on the example from Monday's class.  The youtube video missed the first 30 minutes of this discussion.  This example provides an alternative method for implementing the functions in the homework, but the method that we covered on Monday can also be used for implementing the homework.

I'm happy to schedule additional 1-1 meetings for anyone who has questions about any of the material that didn't make it into the videos.
